help_to_heat/portal/download_views.py

```diff
@@ -279,9 +279,9 @@ def add_extra_row_data(referral, exclude_pii=False):
     created_at = referral.created_at.astimezone(london_tz)
 
     if not exclude_pii:
-        contact_number = row.get("contact_number")
+        contact_number = row.get("contact_number", "")
         contact_number = '="' + contact_number + '"'
-        uprn = row.get("uprn")
+        uprn = row.get("uprn", "")
```

Comment:
Should probably be consistent here with the if/else and defaults. Don't have strong preferences where we go within that.